GoogleForCreators/web-stories-wp
# Pull media action functions out to common

## Summary

Pull media action functions out to common, for reuse in media3p.

## Next Steps

- Add a discriminator to common/reducer.js, so that it can be used for both 'local' and 'unsplash'. See: https://github.com/google/web-stories-wp/pull/2894

## Testing Instructions

- Open the media pane to verify that it appears.
- Scroll down to verify a page of results in returned.
- Upload an image successfully.
- Upload a video successfully.
